### ISSUE TITLE
A11y: Background and foreground colors should have a sufficient contrast ratio

### ISSUE THREAD

**Issue by asavin** [redbadger/website-honestly#365]
**Description of the issue:**

**Browser used:** Chrome
**OS used:**
**Device used:**

**Steps to reproduce:**

1. Install Lighthouse extension for Chrome
2. Run Lighthouse on red-badger.com site and generate full report

**Actual result:**

Following section fails:

Accessibility: Background and foreground colors have a sufficient contrast ratio
Elements must have sufficient color contrast (Failed on 7 elements)

7 elements fail this test learn more
body > .js-app > div > div > div > .style__blogSlice___1NSHc._divider__dividerWhite___2nqlq._divider__divider___2bjqZ > .style__sliceContainer___3sflL > .style__blogLinkList____PSmP > li:nth-of-type(1) > .style__link___NBL04._fonts__fontM___2uO71._fonts__serif___1u07k > .style__category___2pLRW._fonts__fontXS___2mdTk._fonts__boldSansSerif___1G6P5
body > .js-app > div > div > div > .style__blogSlice___1NSHc._divider__dividerWhite___2nqlq._divider__divider___2bjqZ > .style__sliceContainer___3sflL > .style__blogLinkList____PSmP > li:nth-of-type(2) > .style__link___NBL04._fonts__fontM___2uO71._fonts__serif___1u07k > .style__category___2pLRW._fonts__fontXS___2mdTk._fonts__boldSansSerif___1G6P5
body > .js-app > div > div > div > .style__blogSlice___1NSHc._divider__dividerWhite___2nqlq._divider__divider___2bjqZ > .style__sliceContainer___3sflL > .style__blogLinkList____PSmP > li:nth-of-type(3) > .style__link___NBL04._fonts__fontM___2uO71._fonts__serif___1u07k > .style__category___2pLRW._fonts__fontXS___2mdTk._fonts__boldSansSerif___1G6P5
body > .js-app > div > .style__footer___kW08t._divider__dividerBlack___21t8_._divider__divider___2bjqZ > .style__footerContainer___2GIgf > .style__footerEndContainer___1g3L9 > .style__section___C201B.style__disclaimer___2dnWC._fonts__boldSansSerif___1G6P5._fonts__fontXS___2mdTk.style__noBorder___1PgXX > p:nth-of-type(1)
body > .js-app > div > .style__footer___kW08t._divider__dividerBlack___21t8_._divider__divider___2bjqZ > .style__footerContainer___2GIgf > .style__footerEndContainer___1g3L9 > .style__section___C201B.style__disclaimer___2dnWC._fonts__boldSansSerif___1G6P5._fonts__fontXS___2mdTk.style__noBorder___1PgXX > p:nth-of-type(2)
body > .js-app > div > .style__footer___kW08t._divider__dividerBlack___21t8_._divider__divider___2bjqZ > .style__footerContainer___2GIgf > .style__footerEndContainer___1g3L9 > .style__section___C201B.style__disclaimer___2dnWC._fonts__boldSansSerif___1G6P5._fonts__fontXS___2mdTk.style__noBorder___1PgXX > p:nth-of-type(3)
body > .js-app > div > .style__footer___kW08t._divider__dividerBlack___21t8_._divider__divider___2bjqZ > .style__footerContainer___2GIgf > .style__footerEndContainer___1g3L9 > .style__section___C201B.style__disclaimer___2dnWC._fonts__boldSansSerif___1G6P5._fonts__fontXS___2mdTk.style__noBorder___1PgXX > .style__cookieWarning___2pdjL

**Expected result:**

**Comment by asavin** [comment]
<img width="277" alt="home___red_badger" src="https://cloud.githubusercontent.com/assets/487468/22743179/57cf29ea-ee11-11e6-8a71-a8e54672beb8.png">

**Comment by asavin** [comment]
<img width="921" alt="home___red_badger" src="https://cloud.githubusercontent.com/assets/487468/22743251/9639d568-ee11-11e6-9fe0-71815fecfce9.png">

**Comment by gregdorward** [comment]
<img width="375" alt="screen shot 2017-02-08 at 15 54 58" src="https://cloud.githubusercontent.com/assets/20296527/22745033/1756f2ca-ee17-11e6-8f2d-3d0fdecab74d.png">

I think that some of the the contrast issues are caused by the borders around the case studies. Where we have no styled borders, the colour being used is a shade of blue. Although a user can't see this, the contrast analyser must be picking it up

**Comment by gregdorward** [comment]
We need to make the date text either bold, or a different colour, as this is failing accessibility evaluations

<img width="447" alt="screen shot 2017-02-09 at 14 33 17" src="https://cloud.githubusercontent.com/assets/20296527/22787720/27438b08-eed5-11e6-897e-d8481a1af1c5.png">

**Comment by sarigriffiths** [comment]
*Red text*:
In Alex's example "TECHNOLOGY" is in bold font, 16px. It should pass. Upcoming events' dates should be in bold font too. Is that font marked as bold?

**Comment by gregdorward** [comment]
@sarigriffiths no, it's not at the moment, the font family for the dates is "ProximaNova-Light,sans-serif", so that will fail the contrast check as is

**Comment by gregdorward** [comment]
I think this one has been superseded by #431, which contains a resolution to the issue. Are you happy with me closing this one @sarigriffiths?

**Comment by gregdorward** [comment]
We've made a lot of progress with this one, to the extent that this bug no longer reflects the state of our accessibility adherence. We have recently resolved the colour contrasting issues that were the main part of this ticket. I will run a new test soon and raise new tickets if they are needed
